src/lib/one-graph/cli-client.js

```diff
@@ -718,29 +820,80 @@ const generateSessionName = () => {
   return sessionName
 }
 
+/**
+ * Mark a session as inactive so it doesn't show up in any UI lists, and potentially becomes available to GC later
+ * @param {object} input
+ * @param {{metadata: {schemaId:string}; id: string; appId: string; name?: string}} input.session The current session
+ * @param {string} input.netlifyToken The (typically netlify) access token that is used for authentication, if any
+ * @param {NetlifyGraphLockfile.V0_format | undefined} input.lockfile A function to call to set/get the current state of the local Netlify project
+ */
+const maybeUpdateSessionFromLockfile = async (input) => {
+  const { lockfile, netlifyToken, session } = input
+  const sessionSchemaId = session.metadata && session.metadata.schemaId
```

Comment:
We should update the underlying query to get `session.graphQLSchema.id` when it's less painful to do so.